spec/cassettes/OpenTok_Streams/layout_working_on_two_stream_list.yml

```diff
@@ -0,0 +1,36 @@
+---
+http_interactions:
+- request:
+    method: put
+    uri: https://api.opentok.com/v2/project/123456/session/SESSIONID/stream
+    body:
+      encoding: UTF-8
+      string: '{"items":[{"id":"8b732909-0a06-46a2-8ea8-074e64d43422","layoutClassList":["full"]},{"id":"8b732909-0a06-46a2-8ea8-074e64d43423","layoutClassList":["full","focus"]}]}'
+    headers:
+      User-Agent:
+      - OpenTok-Ruby-SDK/3.0.4.alpha1-Ruby-Version-2.5.1-p57
```

Comment:
The Ruby version should be dynamic if not it cause an error with a different version: - OpenTok-Ruby-SDK/<%= version %>